### ISSUE TITLE
Are there auxillary files output similar to salmon?

### ISSUE THREAD

**Issue by callumparr** [COMBINE-lab/oarfish#19]
Does oarfish produce similar auxillary files as salmon? I was particularly interested in the ambig file that tells list number of unique counts, ambiguous counts etc. 

In general for PacBio data, I was following the commands in the supp. of the biorxiv preprint. So use minimap2 with preset map-pb -N 100 and then use oarfish with the coverage information but no filters

```
oarfish --alignments input_bam_file --threads num_threads --output-path output_directory --model-coverage --filter-group no-filters
```

**Comment by rob-p** [comment]
Hi @callumparr,

  Thanks for raising this question.  Currently, there is only one auxiliary file, which records information about the parameters used during quantification and, since the newest release, also records the `seqcol` digest of the reference used for mapping.  Additionally, if you generate inferential replicates with the bootstrap option, those are output as well.

  We currently don't output a file about the unique and maximal possible mappings considered for each transcript.  However, if this is something that you think would be useful for your purposes, please let us know and we'd be happy to consider it as a feature request.  It shouldn't be hard to implement.

Best,
Rob

**Comment by rob-p** [comment]
Hi @callumparr,

  I'm implementing this now on the development branch.  Apart from unique & ambiguous mappings to each transcript, is there other similar information that you'd want `oarfish` to produce?

--Rob

**Comment by callumparr** [comment]
@rob-p Thank you for this. the ambiguous and unique count is all that is needed. TY again.

**Comment by rob-p** [comment]
Sure thing, @callumparr.  This feature is now implemented in 0.5.1 which is released [here](https://github.com/COMBINE-lab/oarfish/releases/tag/v0.5.1) and should also be available via bioconda.

**Comment by callumparr** [comment]
Thank you @rob-p will find the time to pull this and check it out.
